libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationEventDispatcher.java

```diff
@@ -27,13 +27,24 @@
   private CopyOnWriteArrayList<FasterRouteListener> fasterRouteListeners;
   private NavigationMetricListeners.EventListeners metricEventListeners;
   private NavigationMetricListeners.ArrivalListener metricArrivalListener;
+  private RouteUtils routeUtils;
 
   NavigationEventDispatcher() {
     navigationEventListeners = new CopyOnWriteArrayList<>();
     milestoneEventListeners = new CopyOnWriteArrayList<>();
     progressChangeListeners = new CopyOnWriteArrayList<>();
     offRouteListeners = new CopyOnWriteArrayList<>();
     fasterRouteListeners = new CopyOnWriteArrayList<>();
+    routeUtils = new RouteUtils();
```

Comment:
You can replace the first constructor with `NavigationEventDispatcher() { this(new RouteUtils()); }` so you avoid the duplicate code and the chance that the two constructors diverge